src/library/masq.js

```diff
@@ -5,8 +5,10 @@ import swarm from 'webrtc-swarm'
 import pump from 'pump'
 import uuidv4 from 'uuid/v4'
 import { promisifyAll } from 'bluebird'
+import common from '../../node_modules/masq-common/dist/index'
```

Comment:
can you do this instead ? `import common from masq-common/dist/index'`

Reply:
yes, done.